lib/ex_machina/ecto.ex

```diff
@@ -15,8 +15,18 @@ defmodule ExMachina.Ecto do
           ExMachina.Ecto.save_record(__MODULE__, @repo, record)
         end
 
-        defp assoc(attrs, factory_name, opts \\ []) do
-          ExMachina.Ecto.assoc(__MODULE__, attrs, factory_name, opts)
+        defp assoc(_, factory_name, _) do
+          raise """
+          assoc/3 has been removed. Please use build instead.
```

Comment:
Would it be helpful to explain that build associations will be saved when you use `create`?